tox.ini

```diff
@@ -12,17 +12,17 @@ envlist = py34,
 [testenv]
 description = run the test driver with {basepython}
 deps = -rtest-requirements.txt
-commands = python runtests.py -x lint -x self-check {posargs}
+commands = pytest {posargs}
 
 [testenv:lint]
 description = check the code style
 basepython = python3.6
-commands = python runtests.py lint {posargs}
+commands = flake8 -j0 {posargs}
 
 [testenv:type]
 description = type check ourselves
 basepython = python3.6
-commands = python runtests.py self-check -p '-n0' -p '-v'
+commands = python3 -m mypy -p mypy
```

Comment:
For consistency, we should point to the self-check config here.

Reply:
Done